ddcz/models/used/tavern.py

```diff
@@ -22,34 +22,35 @@ class TavernAccessRights(Enum):
 
 
 class TavernTable(models.Model):
-    jmeno = MisencodedCharField(unique=True, max_length=255)
-    popis = MisencodedCharField(max_length=255)
-    vlastnik = MisencodedCharField(max_length=30)
-    povol_hodnoceni = MisencodedCharField(max_length=1)
-    min_level = MisencodedCharField(max_length=1)
-    zalozen = models.DateTimeField()
-    verejny = MisencodedCharField(max_length=1)
-    celkem = models.IntegerField(blank=True, null=True)
-    sekce = models.IntegerField()
+    name = MisencodedCharField(unique=True, max_length=255, db_column="jmeno")
+    description = MisencodedCharField(max_length=255, db_column="popis")
+    owner = MisencodedCharField(max_length=30, db_column="vlastnik")
+    # TODO: MisencodedBooleanField
+    allow_rep = MisencodedCharField(max_length=1, db_column="povol_hodnoceni")
```

Comment:
I do not like allow_rep much, but I don't know what to replace it with, so just a comment. :D

Reply:
Yup, this was the result of the Discord discussion, feel free to amend :)